tests/unit/browser/test_history.py

```diff
@@ -326,6 +326,53 @@ def test_init(self, backend, qapp, tmpdir, data_tmpdir, monkeypatch, cleanup_ini
             assert default_interface is None
 
 
+class TestHistoryClearSite:
```

Comment:
There is a lot of repetition in those tests which could benefit from [parametrization](https://docs.pytest.org/en/stable/how-to/parametrize.html).